src/debug/Stream.h

```diff
@@ -37,10 +37,11 @@
 #define MAX_DEBUG_SECTIONS 100
 
 /* defined names for Debug Levels */
-#define DBG_CRITICAL    0   /**< critical messages always shown when they occur */
-#define DBG_IMPORTANT   1   /**< important messages always shown when their section is being checked */
-/* levels 2-8 are still being discussed amongst the developers */
-#define DBG_DATA    9   /**< output is a large data dump only necessary for advanced debugging */
+#define DBG_CRITICAL   0 /**< critical messages always shown when they occur */
+#define DBG_IMPORTANT  1 /**< important messages always shown when their section is being checked */
+#define DBG_PROTOCOL   2 /**< show flow of protocol messages. Excluding message content/payload which is in DBG_DATA. */
+/* levels 3-8 are still being discussed amongst the developers */
+#define DBG_DATA       9 /**< output is a large data dump only necessary for advanced debugging */
 
```

Comment:
> PR description: add options to debug_options directive for displaying just these traces

Based on the proposed `DBG_PROTOCOL` definition, I would expect that to be already supported by `debug_options ALL,2`. How would those future options differ from that existing support?

Reply:
```ALL,2``` includes a large number of sections that are not about a protocol and print output at level-2. To get a clearer trace right now, admin have to know the individual section numbers which are about protocols and set them to level-2 individually.

The goal for that future work item would be to let admin specify the protocols by name instead of section number. I have a few half-baked ideas so far. Not ready for being in-scope here.

Reply:
To address this change request, I suggest adopting the following wording for the second bullet in the PR TODO list:

* Enhance debug_options to support tracing named protocols

or, a more verbose variant:

* Enhance debug_options to support tracing just the protocol(s) selected by their names.

Reply:
done